pkg/broker/broker.go

```diff
@@ -35,46 +36,61 @@ import (
 
 const registrySecretName = "registry-secret"
 
+//go:generate counterfeiter ./ ClusterFactory
+type ClusterFactory interface {
+	DefaultCluster() (k8s.Cluster, error)
+}
+
+//go:generate counterfeiter ./ HelmClientFactory
+type HelmClientFactory interface {
```

Comment:
Is this the right place for this? I would expect this in `helm` package. Also, the generated doubles aren't check in, so ci would fail.